tasks/register_client.yml

```diff
@@ -81,31 +81,37 @@
     ( 'Failed' in _bconsole_output.stdout )
   changed_when: false
 
-- name: Add backup job
+- name: Create bareos jobdef config files
+  vars:
+    _resource_type: 'JobDefs'
```

Comment:
Is it part of the client registration process?

Reply:
I think it should be. We can't create jobs and jobdefs related to a non-registered client, or `bareos-dir.service` fails. Adding a client without related job makes no sense.
The previous process made no sense because the client registration would happen after the jobs conf files copy. If `bareos-dir` restarts between the templates copy and the client registration, you need to manually delete the config files to be able to start it again.

Reply:
What if one needs to update the configuration after the client is registered?
This is job configuration, it is not strictly related to the client registration process, even if I understand the link between both actions.

Moreover, I used this role in the past to configure new clients and don't remember having issues with `bareos-dir.service`. Is it 100% reproducible? In which case this service will restart while client registration has not completed yet?

Reply:
You're right, we need to be able to update the jobs config without `register_clients: true`.
I guess I had the restart issue because of https://github.com/mila-iqia/ansible-role-bareos/issues/38, but it should disappear once it's fixed.